src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -0,0 +1,43 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Entity(name = "PRODUCTS")
```

Comment:
Co prawda tego to nie ustalaliśmy, ale ja zacząłem nazywać tabele i kolumny w encjach z małej litery. Dla Hibernate to chyba jedno i to samo (dla zapytań w SQL na pewno), ale dla estetyki może nazywajmy małymi? ale jestem podatny na argumenty :)

Reply:
Hehe dla mnie właśnie jak są drukowanymi to widać że to tabele i kolumny bo w IntelliJ pola w klasie są z małej. Mogę zmienić ale to wg mnie odróżnia jedno od drugiego. Co o tym myślisz?

Reply:
to ja pozmieniam u siebie ;) Fakt, że łatwiej odróżnić. I tak się zakręciłem z tym gite-em i finalnie zamknąłem brancha, skopiowałem go i będę robił nowego pull requesta dla encji User.